Lib/asyncio/queues.py

```diff
@@ -167,6 +167,12 @@ def get(self):
                 yield from getter
             except:
                 getter.cancel()  # Just in case getter is not done yet.
+
+                try:
+                    self._getters.remove(getter)
```

Comment:
This is O(n) in the length of the `_getters` deque, which is a bit unfortunate. Perhaps it's fine since this deque should usually be short.